src/main/java/com/aline/accountmicroservice/AccountApplication.java

```diff
@@ -11,9 +11,10 @@
 @EnableJpaRepositories(basePackages = { "com.aline.accountmicroservice.repository", "com.aline.core.repository" })
 @EntityScan(basePackages = { "com.aline.accountmicroservice.model", "com.aline.core.model" })
 public class AccountApplication {
+	public static Integer universalPort = 8069;
```

Comment:
If you're making your port a variable use this:

```suggestion
        @Value("${server.port}")
	public static Integer universalPort;
```